### PR TITLE
Fix the usage of API to retrieve installations and repositories

### PR DESCRIPTION
**Pull request by linsolas**
Hello,

I was facing the issue #35 as other people, so I'm suggesting a small correction to make it work with the latest version of Probot & octokit libraires.

Thanks!

### CODE REVIEW

**Comment by cruzach** [comment]
Hey thanks for sharing this, can confirm it works when you make these exact changes locally in `node_modules`

**Comment by linsolas** [comment]
Hello @cruzach,

Thanks for the confirmation. I will wait few more weeks, but if there is no news on this repo, I may create a fork. Not sure if it is still maintained...

**Comment by cruzach** [comment]
@linsolas let me know if you end creating a fork

**Comment by adamzolyak** [comment]
@linsolas time to fork?

**Comment by adamzolyak** [comment]
@gr2m are you a maintainer for this project.  I saw you merged a bunch of the recent /probot changes.

**Comment by gr2m** [comment]
On probot core, not so much on this plugin. In the Office hours yesterday we decided to archive the scheduler as it's really just a very bad hack, and if you need scheduling, GitHub Actions are a much better approach. If you still need a GitHub App, you can use a GitHub Action cron job to send a request to your app

File: index.js [hunk 1]
```diff
@@ -77,7 +77,8 @@ module.exports = (app, options) => {
     const github = await app.auth()
 
     const installations = await github.paginate(
-      github.apps.listInstallations.endpoint.merge({ per_page: 100 })
+      github.apps.listInstallations({ per_page: 100 }),
```
Comment (gr2m) [inline index.js:80]: I'm a little confused, your change makes it work like it did with Probot 7, passing in an object as `github.apps.listInstallations.endpoint.merge({ per_page: 100 })` returns is the way moving forward

**Comment by adamzolyak** [comment]
@gr2m thanks for the update on the status of this project.  I'll check out using a cron w/ Actions.  Good idea 👏.

For context, my app needs to trigger an Action when an emoji reaction is added to an issue, which isn't currently supported by webhooks.  Thus the need for a cron / scheduler for now.  +1 the [feature request](https://github.community/t5/How-to-use-Git-and-GitHub/Feature-request-webhooks-for-reactions/td-p/23763) for me if you have access internally!

**Comment by adamzolyak** [comment]
Docs for scheduling Actions https://developer.github.com/actions/changes/2019-04-05-scheduling-workflows/

<blockquote><div>GitHub Developer</div><div><strong><a href="https://developer.github.com/actions/changes/2019-04-05-scheduling-workflows/">Scheduling Workflows</a></strong></div><div>Get started with one of our guides, or jump straight into the API documentation.</div></blockquote>

**Comment by linsolas** [comment]
Hello,

Thanks @gr2m for your feedbacks. Github Actions seem indeed to be a better approach, but on my side, I cannot use them right now, since I mainly use a Github Enterprise (on premise), which does not handle (yet) Github Actions. I will keep my internal fork for the moment.

**Pull request closed without merging**